task_schedules/task_schedule_update.cfg

```diff
@@ -0,0 +1,52 @@
+# Configuration file for task_schedule.pl for astromon
+
+subject           astromon update
+timeout           86400             # Default tool timeout
+heartbeat_timeout 1                 # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     /data/aca/astromon/tasks       # Data file directory
+log_dir      /data/aca/astromon/logs  # Log file directory
+heartbeat    update_heartbeat
+#bin_dir     ///     # Bin dir (optional, see task def'n)
+master_log   astromon_update.log             # Composite master log (created in log_dir)
+
+# Email addresses that receive an alert if there was a severe error in
+# running jobs (i.e. couldn't start jobs or couldn't open log file).
+# Processing errors *within* the jobs are caught with watch_cron_logs
+
+alert        javier.gonzalez@cfa.harvard.edu
+notify       javier.gonzalez@cfa.harvard.edu
+
+ notify_msg <<NOTIFY
+  Astromon ran
+NOTIFY
+
+
+<task astromon_update>
+  cron       * * * * *
+  check_cron * * * * *
+  exec 1: cp /data/aca/astromon/ASTROMON_table.h5 /data/aca/astromon/tmp
+  exec 1: astrometry-cat-obs-data --archive-dir /data/aca/astromon/tmp --db-file /data/aca/astromon/tmp/ASTROMON_table.h5
+  exec 1: chown -R :aspect /data/aca/astromon/tmp/obs*
+  exec 1: cp -fr /data/aca/astromon/tmp/obs* /data/aca/astromon/archive
+  exec 1: rm -fr /data/aca/astromon/tmp/obs*
+  exec 1: mv /data/aca/astromon/tmp/ASTROMON_table.h5 /data/aca/astromon/
+  exec 1: astrometry-web-pages --out /proj/sot/ska/www/ASPECT/jgonzalez/celmon/new --db-file /data/aca/astromon/ASTROMON_table.h5
```

Comment:
I will leave this for now, until we publish the web pages and they are in their standard places. I figure this script will change, because we will not want to generate the cal table every time, for example.